README.md

```diff
@@ -260,7 +218,267 @@ EXAMPLES
   $ aio app state stats --json
 ```
 
-## `aio app db ping`
+# Database Commands
+
+## Collection Management
+
+### `aio app db collection create COLLECTIONNAME`
+
+Create a new collection in the database
+
+```
+USAGE
+  $ aio app db collection create COLLECTIONNAME [--json] [--region amer|emea|apac] [-c <value>] [-v <value>]
+
+ARGUMENTS
+  COLLECTIONNAME  The name of the collection to create
+
+FLAGS
+  -c, --collation=<value>  Collation for text comparison and sorting (e.g., "en_US", "simple")
+  -v, --validator=<value>  JSON schema validator for document validation (JSON string)
+      --region=<option>    State region. Defaults to 'AIO_STATE_REGION' env or 'amer' if neither is set.
```

Comment:
This is not a collection level option. Collections are always in the same region as the database.